libmamba/include/mamba/core/link.hpp

```diff
@@ -22,10 +23,12 @@
 
 namespace mamba
 {
+    class Context;
```

Comment:
I don't think we need this one here (I don't see any usage of `Context` in this header)

Reply:
There is probably a lot of these leftovers from when I changed apis one way and then another way. I'll try to do a cleanup. 👍🏽